portals/publisher/source/src/app/components/Apis/Details/Configuration/components/UpdateWithoutDetails.jsx

```diff
@@ -0,0 +1,261 @@
+import MuiDialogContent from '@material-ui/core/DialogContent';
+import MuiDialogActions from '@material-ui/core/DialogActions';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+import Typography from '@material-ui/core/Typography';
+import { FormattedMessage } from 'react-intl';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import Divider from '@material-ui/core/Divider';
+import TextField from '@material-ui/core/TextField';
+import API from 'AppData/api';
+
+const styles = (theme) => ({
+    root: {
+        margin: 0,
+        padding: theme.spacing(2),
+    },
+    closeButton: {
+        position: 'absolute',
+        right: theme.spacing(1),
+        top: theme.spacing(1),
+        color: theme.palette.grey[500],
+    },
+});
+
+const DialogTitle = withStyles(styles)((props) => {
+    const {
+        children, classes, onClose, ...other
+    } = props;
+    return (
+        <MuiDialogTitle disableTypography className={classes.root} {...other}>
+            <Typography variant='h6'>{children}</Typography>
+            {onClose ? (
+                <IconButton aria-label='close' className={classes.closeButton} onClick={onClose}>
+                    <CloseIcon />
+                </IconButton>
+            ) : null}
+        </MuiDialogTitle>
+    );
+});
+
+const DialogContent = withStyles((theme) => ({
+    root: {
+        padding: theme.spacing(2),
+    },
+}))(MuiDialogContent);
+
+const DialogActions = withStyles((theme) => ({
+    root: {
+        margin: 0,
+        padding: theme.spacing(1),
+    },
+}))(MuiDialogActions);
+
+/**
+ *
+ * @returns
+ */
+export default function UpdateWithoutDetails(props) {
+    const {
+        classes, api, apiConfig, handleClick, open, handleClose,
+    } = props;
+
+    let availableEndpoint = '';
+    let endpointType = 'PRODUCTION';
+    if (apiConfig.endpointConfig) {
+        if (apiConfig.endpointConfig.production_endpoints) {
+            availableEndpoint = apiConfig.endpointConfig.production_endpoints.url;
+            endpointType = 'PRODUCTION';
+        } else if (apiConfig.endpointConfig.sandbox_endpoints) {
+            availableEndpoint = apiConfig.endpointConfig.sandbox_endpoints.url;
+            endpointType = 'SANDBOX';
+        }
+    } else if (api.advertiseInfo && api.advertiseInfo.apiExternalProductionEndpoint) {
+        availableEndpoint = api.advertiseInfo.apiExternalProductionEndpoint;
+        endpointType = 'PRODUCTION';
+    } else if (api.advertiseInfo && api.advertiseInfo.apiExternalSandboxEndpoint) {
+        availableEndpoint = api.advertiseInfo.apiExternalSandboxEndpoint;
+        endpointType = 'SANDBOX';
+    }
+
+    const [endpointUrl, setEndpointUrl] = useState(availableEndpoint);
+    const [isValidEndpoint, setValidEndpoint] = useState(availableEndpoint !== '');
```

Comment:
remove this state

Reply:
this was kept to keep the validation state, since otherwise it should be calculated all the time